repl/repl.go

```diff
@@ -1170,6 +1213,7 @@ var builtin = [...]commandDesc{
 	{"json", []string{}, "set output format to JSON"},
 	{"pretty", []string{}, "set output format to pretty"},
 	{"trace", []string{}, "toggle full trace"},
+	{"metrics", []string{}, "toggle profiling"},
```

Comment:
The `metrics` command would be great! Please pull that out into a separate PR. We can probably merge it right away.